src/scss/includes/components/panel.scss

```diff
@@ -13,13 +13,15 @@
   background-color: white;
   width: $panel_width;
 
-  &-header {
-    padding: 0 15px;
-    min-height: 40px;
-    color: #5c6f84;
-    span {
-      width: 100%;
-    }
+  &-drawer {
```

Comment:
According to the design, this drawer should be 20px high. Here we see there isn't enough vertical space between the panel top and the content.

Reply:
Yep, I remove at one time trying to fix an issue. I fixed it, thanks